os/drivers/lcd/lcd_dev.c

```diff
@@ -461,16 +465,20 @@ int lcddev_register(struct lcd_dev_s *dev)
 	lcd_info->pm_domain = pm_domain_register("LCD");
 #endif
 
-	lcd_init_put_image(dev);
 	sem_init(&lcd_info->sem, 0, 1);
-	if (dev->setpower) {
+	bool is_silent_mode;
+	is_silent_mode = silent_reboot_is_silent_mode();
+	if (!is_silent_mode) {
+		ASSERT(dev->setpower);
 		ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
+#ifdef CONFIG_PM
+                (void)pm_suspend(lcd_info->pm_domain);
+#endif
+		silent_reboot_lock();
+		lcd_init_put_image(dev);
 		if (ret != OK) {
 			goto cleanup;
```

Comment:
if it calls here, we need to add pm_reusm() and silent_reboot_unlock into cleanup:
Let's move it before pm_suspend calling